src/containers/Accounts/index.js

```diff
@@ -23,12 +23,14 @@ const Accounts = ({
   isLastPage,
   isAuth0Enabled,
   allUserRoles,
-  extraFields,
+  formModal,
 }) => {
   const [editModal, setEditModal] = useState(false);
   const [addModal, setAddModal] = useState(false);
   const [activeUser, setActiveUser] = useState({});
 
+  const FormModal = formModal || DefaultForm;
```

Comment:
if you set the defaultProp for this to DefaultForm will it remove the need for this?